README.md

```diff
@@ -17,3 +17,7 @@ or conda-forge:
 ```
 conda install -c conda-forge jupyter_events
 ```
+
+## Documentation
+
+Documentation is available at [jupyterlab.readthedocs.io](https://jupyterlab.readthedocs.io).
```

Comment:
Should this be https://jupyter-events.readthedocs.io/en/latest/?

Reply:
No—I mean this should be the Jupyter Events link... not JupyterLab.

Reply:
My comment wasn't as petty as you read :wink:

Reply:
You don't have to include the `en/latest`. Sorry, that was added by my copy and paste actions... haha

Reply:
OMG. 😂 Right, sorry. Fixed the link.